repos/spack_repo/builtin/packages/py_b2luigi/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyB2luigi(PythonPackage):
+    """b2luigi is a helper package constructed around luigi that helps you schedule working
+    packages (so-called tasks) locally or on a batch system. Apart from the very
+    powerful dependency management system by luigi, b2luigi extends the user interface
+    and has a built-in support for the queue systems, e.g. LSF and HTCondor."""
+
+    homepage = "https://github.com/belle2/b2luigi"
+    pypi = "b2luigi/b2luigi-1.2.6.tar.gz"
+
+    license("GNU")
+
```

Comment:
I know this is not technically HEP-only, but it is definitely a project from within the HEP  community. Adding a tag helps us (HEP-ecosystem maintainers) keep an eye on which of the `hep`-tagged packages can be updated.

```suggestion

    tags = ["hep"]

```

Automatic CI builds can be added to spack CI as well; see https://github.com/spack/spack-packages/blob/develop/stacks/hep/spack.yaml.

Reply:
Thank you @wdconinc I had not come across these tags this is good to know